canvas_modules/harness/webpack.config.dev.js

```diff
@@ -120,6 +125,13 @@ module.exports = {
 			__dirname,
 			"node_modules"
 		],
+		fallback: {
+			// path: require.resolve("path-browserify"),
+			// stream: require.resolve("stream-browserify"),
+			util: require.resolve("util"),
+			buffer: require.resolve("buffer"),
+			vm: false
```

Comment:
@matthoward366 Carbon 11 works on Webpack 5.x. I tried upgrading test harness webpack from 4.x to 5.x in commit - https://github.com/elyra-ai/canvas/pull/1704/commits/33cc657ea275e3d4adb437305b8f7ae77ef82416

Referred to this canvas-ui PR - https://github.ibm.com/NGP-TWC/wml-canvas-ui/pull/3205/files

But when I run `npm start` in test harness, I see this error - 
```
ValidationError: Invalid options object. Dev Middleware has been initialized using an options object that does not match the API schema.
 - options has an unknown property 'noInfo'. These properties are valid:
   object { mimeTypes?, mimeTypeDefault?, writeToDisk?, methods?, headers?, publicPath?, stats?, serverSideRender?, outputFileSystem?, index?, modifyResponseData? }
```

Do you know how to fix this error?